package pt.tecnico.sise.se.insure.domain.exception;

public class NIFAlreadyInUseException extends RuntimeException {

}
